EstateReporting.IntegrationTests/Common/DockerHelper.cs

```diff
@@ -0,0 +1,503 @@
+﻿namespace EstateReporting.IntegrationTests.Common
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Data;
+    using System.IO;
+    using System.Linq;
+    using System.Net;
+    using System.Net.Http;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Client;
+    using Database;
+    using Ductus.FluentDocker.Builders;
+    using Ductus.FluentDocker.Common;
+    using Ductus.FluentDocker.Model.Builders;
+    using Ductus.FluentDocker.Services;
+    using Ductus.FluentDocker.Services.Extensions;
+    using EstateManagement.Client;
+    using EventStore.Client;
+    using global::Shared.Logger;
+    using Microsoft.Data.SqlClient;
+    using SecurityService.Client;
+    using TransactionProcessor.Client;
+
+    /// <summary>
+    /// 
+    /// </summary>
+    /// <seealso cref="Shared.IntegrationTesting.DockerHelper" />
+    public class DockerHelper : global::Shared.IntegrationTesting.DockerHelper
+    {
+        #region Fields
+
+        /// <summary>
+        /// The estate client
+        /// </summary>
+        public IEstateClient EstateClient;
+
+        /// <summary>
+        /// The security service client
+        /// </summary>
+        public ISecurityServiceClient SecurityServiceClient;
+
+        /// <summary>
+        /// The test identifier
+        /// </summary>
+        public Guid TestId;
```

Comment:
Codacy found an issue: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/manual/stuart_ferguson1/EstateReporting/pullRequest?prid=6339123)